CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bug Fixes
 1. [17039](https://github.com/influxdata/influxdb/pull/17039): Fixed issue where tasks are exported for notification rules
 1. [17042](https://github.com/influxdata/influxdb/pull/17039): Fixed issue where tasks are not exported when exporting by org id
+1. [17050](https://github.com/influxdata/influxdb/pull/17050): Fixed the CLI auth find, user name issue
```

Comment:
This isn't very clear, maybe something like "Added missing user names to auth CLI commands"